tools/explorer/pkg/stellar/stellar.go

```diff
@@ -292,6 +292,7 @@ func (w *Wallet) GetBalance(address string, id schema.ID, asset Asset) (xdr.Int6
 	txReq := horizonclient.TransactionRequest{
 		ForAccount: address,
 		Cursor:     cursor,
+		Limit:      200,
```

Comment:
lets make this a constant, because you will also need to change the condition of the for loop below. rather than checking if the txes are empty, you'll need to check if the amount of txes is smaller than the limit